src/controller.js

```diff
@@ -4,6 +4,14 @@ function timeout(ms) {
     return new Promise(resolve => setTimeout(resolve, ms));
 }
 
+HACK_ON = `
```

Comment:
1. I moved the string outside of the main controller class because I wanted no spaces between the left edge of the ascii art and the line start. When the text gets printed in the terminal it goes from `      Hack On` to `Hack On`. 

2. The ascii art itself is almost unrecognizable. This is because I had to manually escape all the `\` characters. By default Javascript will interoperate characters differently when they are  proceeded with a `\`. For example, `n` is just the letter `n` but `\n`' is a special way to denote the "new line character". In this case, the `\`s were escaping random things that didn't render to any visible character so the spacing was getting messed up b/c `\a` takes up 2 characters in the string but gets rendered as 1. Additionally, the `\` at the end of the line acts as a way to negate the new line. Therefore, your code which attempted to check if a character was a new line would never find that character because it was getting  escaped. 

There are a couple ways of dealing with this. We could read in the file from an external source over HTTP, [bundle the raw file into the javascript output using a browserify plugin](https://github.com/JohnPostlethwait/stringify) or escaping that characters inline. That's what I chose to do because it was the easiest/simplest.